addons/binding/org.openhab.binding.spotify/README.md

```diff
@@ -0,0 +1,114 @@
+# OpenHAB2 binding for Spotify 
+
+This binding implements a bridge to the Spotify Player Web API and makes it possible to discover Spotify Connect Devices on your network using you Spotify Premium account.
+
+The binding requires you to register an Application with Spotify Web API on https://developer.spotify.com 
+
+When registering your new Spotify Application for OpenHAB Spotify Bridge you have to specify the allowed "Redirect URIs" aka white-listed addresses. Here you have to specify the address to you Eclipse Smarthome server. 
+
+If you run it on the local machine with default settings you add "http://localhost:8080/connectspotify/authorize" to the Redirect URIs. The authentication process takes place using your client web browser so this Redirect URI is where the OpenHAB web UI is.
+
+Steps required to get this binding running:
+0. Install bind and make sure the Spotify Bridge things are available on your server
+1. Register on https://developer.spotify.com with your Spotify Premium Account
+2. Add a new Application. Add Redirect URI for OpenHAB. We need the "Client ID" and "Client Secret" to add a new binding.
```

Comment:
Many thanks for all your review and feedback! 

I will take care of that as soon as possible. I already started changes on the channels and categorized advanced ones. I have an updated README ready as well. I also  noticed some polling thread issue which I didn't  manage to solve and push yesterday. Won't have time until Sunday evening to work on this again.